dependencies/src/test/resources/RepositoryUtilityTestGoldenBomArtifacts.txt

```diff
@@ -0,0 +1,223 @@
+# Automatically generated by running tests with environment variable
```

Comment:
I strongly prefer not to check any auto generated files into the repo. That's a big antipattern. We can autogenerate files once or every time. In this case, I think it makes sense to do it once and then manually edit it from there on in.

Reply:
This is a common pattern for golden file tests. I think it's helpful to make this test easy to maintain, if we want the test at all (see below).

Reply:
No uneditable code in the repo. That's a hard line for me. I've bit my tongue and let this go in the past in the interests of immediate convenience -- e.g. in Cloud Tools for Eclipse target platform definitions -- and I've always ended up regretting it.

Reply:
Up to you.